src/betterproto/plugin/models.py

```diff
@@ -334,6 +335,22 @@ def deprecated_fields(self) -> Iterator[str]:
     def has_deprecated_fields(self) -> bool:
         return any(self.deprecated_fields)
 
+    @property
+    def has_oneof_fields(self) -> bool:
+        return any(
+            map(lambda field: isinstance(field, OneOfFieldCompiler), self.fields)
+        )
```

Comment:
```suggestion
        return any(
            isinstance(field, OneOfFieldCompiler) for field in self.fields
        )
```
not sure how I missed this first time round